src/components/dashboard.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import './dashboard.css';
+
+const Dashboard = () => {
```

Comment:
Will this component be implemented later on? If so, it's stylesheet should not be re-used by other components. It should have it's own stylesheet, and the other components must have their own stylesheets as well